src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -253,6 +253,16 @@ export class StudyViewPageStore {
         }
     }
 
+    @observable.ref private _comparisonConfirmationModal:JSX.Element|null = null;
```

Comment:
is there a reason to bother with get/set when we can't use set (i guess because of the decorator problem)?

Reply:
hm. this is weird.  not sure what this does

Reply:
I can see why it might be weird. i just thought it was syntactically sensible  since the getter doesn't do any computation. the actual reason we cant use a setter is because the setters argument structure is very constrained